code/modules/mob/living/carbon/human/human_say.dm

```diff
@@ -1,5 +1,6 @@
 /mob/living/carbon/human/say_mod(input, list/message_mods = list())
-	verb_say = dna.species.say_mod
+	var/obj/item/organ/tongue/T = src.getorganslot(ORGAN_SLOT_TONGUE)
+	verb_say = T.say_mod
```

Comment:
Can we cache this to a var on /carbon and have it be set on organ insert?

Reply:
Added to to-do

Reply:
That's already a thing.

```getorganslot(organ_slot)``` returns the value of an assoc list at index ```organ_slot```.

The assoc list is already a part of /carbon and is updated whenever an organ is inserted into a specific slot.

Reply:
```/mob/living/carbon/getorgan(typepath)
	return (locate(typepath) in internal_organs)```
	
	this is stinky

Reply:
That's not what is used.

```
/mob/living/carbon/getorganslot(slot)
	return internal_organs_slot[slot]
```
is how any tongue is being referenced for say_mods.

Reply:
> ah. Why the fuck do we have so many organ helpers jesus christ

Something something not knowing about proc overhead or other

Reply:
If you don't acknowledge the technical debt, it cannot hurt you